torchci/rockset/metrics/__sql/master_commit_red_avg.sql

```diff
@@ -63,21 +35,26 @@ reds AS (
                 END
             ) > 0 AS int
         ) AS any_red,
-        COUNT(*)
+        CAST(
+            SUM(
+                CASE
+                    WHEN conclusion = 'failure' AND attempt = 1 THEN 1
```

Comment:
`attempt = 1` isn't a super reliable filter.  On any PR that requires a someone to hit the "Approve" button for CI to run, the first attempt is `attempt = 2` (there will be no attempt #1)